src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -329,14 +335,7 @@ public void onTraversalStart() {
         );
 
         // Sort our data source factories to ensure they're always in the same order:
-        dataSourceFactories.sort( Comparator.comparing(DataSourceFuncotationFactory::getName) );
-
-        // Identify and store any Gencode data sources:
-        for ( final DataSourceFuncotationFactory factory : dataSourceFactories ) {
-            if ( factory.getType().equals(FuncotatorArgumentDefinitions.DataSourceType.GENCODE) ) {
-                gencodeFuncotationFactories.add( (GencodeFuncotationFactory) factory );
-            }
-        }
+        dataSourceFactories.sort(DataSourceUtils::datasourceComparator);
```

Comment:
Can you add to this comment that gencode factories will always be first as well?

Reply:
Done